README.md

```diff
@@ -53,61 +66,82 @@ The mediator is especially useful when the edge entities are not always online,
+You can open the `http://localhost:8080/` URL in a web browser, and it will show a QR code that serves as an out-of-band invitation for the Mediator.
 
-**Open the webpage for develop** - open> `file:///.../webapp/index-fastopt.html`
+## How to run mediator as docker image
+# Docker only
+Everything can be run with a single command with Docker compose docker-compose.yml
+The latest stable image version can also be downloaded from the IOHK repositories.
+To build to docker image locally run  NODE_OPTIONS=--openssl-legacy-provider sbt docker:publishLocal.
+```
+shell> cd atala-prism-mediator
+shell> NODE_OPTIONS=--openssl-legacy-provider sbt docker:publishLocal
+shell> docker compose up
+```
 
 ### Configure the Mediator
 
 The default configuration is set up [application.conf](/mediator/src/main/resources/application.conf).
 So in order to configure the mediator for your needs.
 You can either change the default configuration or you can set up environment variables that overrides the defaults:
 
+# identity
 To set up the mediator identity:
 - `KEY_AGREEMENT_D` - is the key agreement private key (MUST be a X25519 OKP key type).
 - `KEY_AGREEMENT_X` - is the key agreement public key (MUST be a X25519 OKP key type).
 - `KEY_AUTHENTICATION_D` - is the key authentication private key (MUST be an Ed25519 OKP key type).
 - `KEY_AUTHENTICATION_X` - is the key authentication public key (MUST be an Ed25519 OKP key type).
 - `SERVICE_ENDPOINT` - is the endpoint of the mediator. Where the mediator will be listening to incoming DID Comm messages.
 
+# mediator-storage
 To set up the mediator storage (MongoDB):
 - `MONGODB_PROTOCOL` - is the protocol type used by mongo.
 - `MONGODB_HOST` - is the endpoint where the MongoDB will be listening.
 - `MONGODB_PORT` - is the endpoint's port where the MongoDB will be listening.
-- `MONGODB_USER` - is the user name used by the Mediator service to connect to the database.
+- `MONGODB_USER` - is the username used by the Mediator service to connect to the database.
 - `MONGODB_PASSWORD` - is the password used by the Mediator service to connect to the database.
 - `MONGODB_DB_NAME` - is the name of the database used by the Mediator.
 
-## Run
-
-This DIDComm Mediator is composed of two elements, a backend service, and a database.
-The backend service is a JVM application and the database used is MongoDB.
-The backend service is also a web service that has a single-page application that will give the final user an invitation page.
-
-### Run localy
+### MongoDB In cloud
 
-Everything can be run with a single command with Docker compose `docker-compose.yml`
-
-First build to docker image with `NODE_OPTIONS=--openssl-legacy-provider sbt docker:publishLocal`.
-The latest stable image version can also be downloaded from the IOHK repositories.
-
-### MongoBD
-
-Docker compose would do that for you but if you are running separately or on the cloud like MongoDB Atlas.
+Using the mongodb from cloud like MongoDB Atlas.
```

Comment:
This is slightly unclear; I believe the intention is something like this maybe?
Use the mongodb from the cloud, similar MongoDB Atlas.

Reply:
@petevielhaber This is if a user wants to use MongoDB from the cloud instead of running in MongoDB as a docker 

Running MongoDB in a production environment might involve setting up the cluster and having backup requirements etc, they can use the cloud version which is a managed service for MongoDB and Atlas in one of the e.g. 
Hope that helps Let me know if that helps